s3/pkg/service/object.go

```diff
@@ -289,10 +289,17 @@ func (s *s3Service) GetObjectMeta(ctx context.Context, in *pb.Object, out *pb.Ge
 
 func (s *s3Service) GetObject(ctx context.Context, req *pb.GetObjectInput, stream pb.S3_GetObjectStream) error {
 	log.Infoln("GetObject is called in s3 service.")
+	var versionId string
 	bucketName := req.Bucket
 	objectName := req.Key
 	offset := req.Offset
 	length := req.Length
+	if req.VersionId != ""{
```

Comment:
Is it necessary to define versionId, is there any difference from versionId and req.VersioinId?